release-notes/opensearch-ml-common.release-notes-2.14.0.0.md

```diff
@@ -12,22 +12,26 @@ Compatible with OpenSearch 2.14.0.0
 * Migrate RAG pipeline to async processing. ([#2345](https://github.com/opensearch-project/ml-commons/pull/2345))
 * Filtering hidden model info from model profiling for users other than superadmin ([#2332](https://github.com/opensearch-project/ml-commons/pull/2332))
 * check model auto deploy  ([#2288](https://github.com/opensearch-project/ml-commons/pull/2288))
-* restrict stash context only for stop words system index (https://github.com/opensearch-project/ml-commons/pull/2283)
-* Add a flag to control auto-deploy behavior (https://github.com/opensearch-project/ml-commons/pull/2276)
+* restrict stash context only for stop words system index ([#2283](https://github.com/opensearch-project/ml-commons/pull/2283))
```

Comment:
Is this really an enhancement or bug fix?

Reply:
I think it's more like an enhancement for better security purposes.